src/modules/jcms_rest/tests/src/Unit/JCMSImageUriTraitTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\Tests\jcms_rest\Unit;
+
+use Drupal\Core\Site\Settings;
+use Drupal\jcms_rest\JCMSImageUriTrait;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests for methods in JCMSImageUriTrait.
+ *
+ * @package Drupal\Tests\jcms_rest\Unit
+ */
+class JCMSImageUriTraitTest extends UnitTestCase {
+
+  use JCMSImageUriTrait;
```

Comment:
I'd say that mixing production code with tests shows that `JCMSImageUriTrait` needs to be refactored, but this is better than nothing.